src/scripts/seedDB.js

```diff
@@ -0,0 +1,171 @@
+require("dotenv").config({ silent: true });
+
+const { DB_URL } = process.env;
+
+// db dependencies
+const mongoose = require("mongoose");
+const uuid = require("uuid");
+
+// validation dependencies
+const { isValid, format } = require("date-fns");
+const bcrypt = require("bcryptjs");
+const { convert } = require("emoji-text");
+
+// scraping dependecies
+const cheerio = require("cheerio");
+const axios = require("axios");
+
+const fetch_url = "https://mlh.io/seasons/na-2019/events";
+const seedInstance = axios.create({ baseURL: fetch_url });
+
+/** START DB Connection */
+if (!DB_URL) {
+  console.log("> 🙅🏻‍♂️ DB URL not provided");
+  process.exit(0);
+}
+
+const options = {
+  useNewUrlParser: true,
+  useCreateIndex: true,
+  useFindAndModify: false
+};
+
+async function db() {
+  Promise.resolve(mongoose.connect(DB_URL, options));
```

Comment:
Kinda sucks that we didn't reuse the connection the app already has. We're basically doing the same thing twice.